examples/env_filter.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 FastLabs Developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use logforth::append;
+use logforth::filter::EnvFilter;
+use logforth::layout::TextLayout;
+use logforth::Dispatch;
+use logforth::Logger;
+
+fn main() {
```

Comment:
It may be better if you add some command line example for different env filter cases.

Reply:
Things like `RUST_LOG=info cargo run --features="env-filter" --example env_filter` ..

I don't find a place suitable for this and would prefer to postpone it as a follow-up.